pandas/tests/window/test_rolling.py

```diff
@@ -446,3 +446,38 @@ def test_min_periods1():
     result = df["a"].rolling(3, center=True, min_periods=1).max()
     expected = pd.Series([1.0, 2.0, 2.0, 2.0, 1.0], name="a")
     tm.assert_series_equal(result, expected)
+
+
+@pytest.mark.parametrize("test_series", [True, False])
```

Comment:
Can you just parameterize over the constructor and use `tm.assert_equal` at the end?

Reply:
done